src/components/Shared/UI/DTESelect/DTESelect.tsx

```diff
@@ -48,7 +63,7 @@ const DTESelect = memo(
         key={option.value}
         data-testid={option?.testID}
       >
-        {option.text}
+        {capitaliseWords(option.text)}
```

Comment:
conditionally capitalise based off boolean to stop this effecting other places the DTESelect is being use.

so define the type:
```
type Props ={
  otherProps
  isCapitalised?: bool
}
```
adding the ? makes it an optional type

Then pass it in the props with a default value of false
`isCapitalised = false`
setting this to false by default will mean the ternary expression will return the non-capped version of words therefore not changing any behaviour elsewhere

finally use it in a ternary expression which is simply shorthand for if/else
`{isCapitalised ? capitaliseWords(option.text) : option.text}